test/support/rails_application_fixture_helper.rb

```diff
@@ -0,0 +1,75 @@
+# typed: ignore
+# frozen_string_literal: true
+
+require "rails_test_helper"
+
+module RailsApplicationFixtureHelper
+  include ApplicationFixtureHelper
+
+  def setup_application_fixture
+    super()
+    cache_rails_paths
+  end
+
+  def teardown_application_fixture
+    restore_rails_paths
+    super()
+  end
+
+  def copy_template(template)
+    super(template)
+
+    case template
+    when :minimal
+      rails_autoload_paths(paths_to("/components/sales/app/models"))
+      rails_eager_load_paths([])
+      rails_autoload_once_paths([])
+      rails_root_path(app_dir)
+    when :skeleton
+      rails_autoload_paths(paths_to("components/sales/app/models"))
+      rails_eager_load_paths(paths_to("components/platform/app/models"))
+      rails_autoload_once_paths(
+        paths_to(
+          "components/timeline/app/models",
+          "components/timeline/app/models/concerns",
+          "vendor/cache/gems/example/models",
+        )
+      )
+      rails_root_path(app_dir)
```

Comment:
Shouldn't `rails_root_path(app_dir)` apply to any template we use?

Reply:
It does.  I chose to repeat this line in both `when` blocks rather than extract it at the bottom.  If I shorten the method by extracting the path setting logic into separate methods, then I can have a single `rails_root_path(app_dir)` outside of the `case` statement.